tests/bal-tools-integration-tests/src/test/java/org/ballerinalang/test/tools/InBuiltToolCommandsTests.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright (c) 2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets use LLC header

Reply:
Addressed in [0051112](https://github.com/ballerina-platform/ballerina-lang/pull/44104/commits/0051112995b8aa8fd9bb7df0099b63fe94e3a5b1)